src/layouts/ConnectionLayout.astro

```diff
@@ -1,18 +1,25 @@
 ---
-import BaseLayout from './BaseLayout.astro';
-import Farmlands from '../components/Farmlands.vue';
-import GuideLayout from '../components/GuideLayout.vue';
----
+import BaseLayout from '../layouts/BaseLayout.astro';
+import Prose from '../components/Prose.vue';
 
-<BaseLayout>
-  <Farmlands/>
-    <GuideLayout
-      collectionName=""
-      headings={{}}
-      title="Overview"
-      img=""
-      fullWidth={true}
-      >
-    <slot />
-  </GuideLayout>
+const { title, description, img } = Astro.props;
+---
+<BaseLayout title={title} description={description} img={img}>
+  <div>
+    <slot name="hero" />
```

Comment:
Why is this slot named hero? I think it would be better named header. I think that is what it is described as elsewhere in the codebase.

Reply:
I think hero is a more descriptive word than header but I can change it

"A hero image is a website design term used to describe an oversized banner image at the top of a website. Sometimes called a “hero header”, it serves as a user’s first glimpse of your company and offering because of its prominent placement towards the top of a webpage that usually extends full-width." - https://www.optimizely.com/optimization-glossary/hero-image/#:~:text=A%20hero%20image%20is%20a,that%20usually%20extends%20full%2Dwidth.

Reply:
I know the term I just feel that it might be too specific as it may not nesicarily be a hero header. I'm easy.